cluster-autoscaler/core/utils.go

```diff
@@ -49,6 +49,9 @@ import (
 const (
 	// ReschedulerTaintKey is the name of the taint created by rescheduler.
 	ReschedulerTaintKey = "CriticalAddonsOnly"
+	// MaxSchedulableLog is the maximum number of schedulable pods for which
+	// we will log information where they could fit for verbosity < 10.
+	MaxSchedulableLog = 5
```

Comment:
I would log full output at V5 or V6. It will still be off by default, but you can change it if you need logs for a specific pod, without getting ridiculous amount of predicate logs at V10.

Also I would set the threshold higher. Each pod logs at maximum 6 lines / minute, so 20 would still keep us around 100 lines / minute and it's more likely to give a better sample for debugging.

Reply:
20 or more.